pkg/services/search/sorting.go

```diff
@@ -0,0 +1,48 @@
+package search
+
+import (
+	"github.com/grafana/grafana/pkg/services/sqlstore/searchstore"
+	"sort"
+)
+
+var (
+	sortAlphaAsc = SortOption{
+		Name:        "alpha-asc",
+		DisplayName: "A-Z",
+		Description: "Sort results in an alphabetically ascending order",
+		Filter:      searchstore.TitleSorter{},
+	}
+	sortAlphaDesc = SortOption{
+		Name:        "alpha-desc",
+		DisplayName: "Z-A",
+		Description: "Sort results in an alphabetically descending order",
+		Filter:      searchstore.TitleSorter{Descending: true},
+	}
+)
+
+type SortOption struct {
+	Name        string
+	DisplayName string
+	Description string
+	Filter      searchstore.FilterOrderBy
+}
+
+var sortOptions = map[string]SortOption{
+	sortAlphaAsc.Name:  sortAlphaAsc,
+	sortAlphaDesc.Name: sortAlphaDesc,
+}
+
+func RegisterSortOption(option SortOption) {
```

Comment:
Can we avoid adding this function since it's not in use? I'd prefer not having stale code sitting around.

Reply:
It's in use, but not in this repo

Reply:
(I'll add doc comments for this)

Reply:
Aha, thanks @sakjur.